notebooks/249-oneformer-segmentation/249-oneformer-segmentation.ipynb

```diff
@@ -0,0 +1,577 @@
2KKe8eGZ0bmxaA/vjTvVkaWFu5zd+93/9/u//6d0vv7rxVvWwWedyn5oakUfUPz2/pEfJaeegtn5wTz8p/tJyobi+uYVOh0Sl9NCuTLYMiy3PIULrcaAUfyEhRRrGcolA9g+jaE0+Bg8w9U/RnOP2j96hgOLchbP8kCCqGCvZjjpeZjjesUiJx3Ozixx6Mg0khN67+4ABt7JyuLy8smniBe+SmrFRCS8kx1CzcajQIxSFXMYUMsV5BsYhRmqvYcbzowUkJkOFDFh9cF/HR8exqyP/fF6W2vnzFWxZtRI15qC6iRdlh7LV/U0dBzvdmnQ9pmGpZEditEPGIh0fdXRHUkEOH0jZ/dMDKsj1R58w7n5yPMcp6piCaYwmtMHOHRBSb7V9CGMdq3O3L33KIIv5RYEC0W2KERzBjuHJPoce5cCv8JknOG1EvsihBoxOuap2XzIBjiFLcSI8iC5j27GSSSPIiSo43hudfUKm1thUz2oLMoHvr62yZNAGguF7d8owB9KCBmyRI+qGNmWpbohvIW94RWbS3awhOyUW0GLk0zVRgfkRuHf4b5A5vbBPM40XknIQxqpA9QHmoz044J7UAk16uDmieDC8rHTKI52WgmNw5HbafKWo/+LFS7//hT9gRQzVT5STsdT51A/26pju9u4eV9s7zlyj5UISw8sr0/NWZfuVsgSXGQ9tVA9ESfj/XM8Tw5pOTCP4NoK1DPEL1+t+ga9ItQuvUlSpoPtDsQaOEycFPpqdwUk7cn9+BA003Qe/wmFcKXSA6/LjofVAhORydzoileqzAMhKPM5XrNO3WOoSs6PIhT0FEMwUPkFuE44mj3FHmjRewN5zBjQv4sJN8RdyO1zb9aoHYwRe3/VnvwdC375/J1gNI7/ZYv56c/vNG4z1i1evgLi0bwLhxptvog886B2PP/6VL3/59luvdxrVSjm3uzn81378r2g88ku/+kukvtirbtR4qJuLU86WF6Ll5hAHSKe19rCR+vRqb/mNV17+9V//9SsXLwEilEKozCYZC7bNlKJuWIkyjP/Pz/7Mpz/96d/4jd9YfRidyahs/92P/zUD/oyZvHb5is1p/wYufdpaGPpDtIZSRL7lPSpZHpkaLwkRcWIENHTMoG5OjGmeQmARqxIfXMaiG/C7OAlZZ1M5GOYQEs4N86NRZkET7TU7GvoIRkYANaLaUToZ3moHx1MRbLcbJUOQQOK6r9MhwDmkBf5L4JxGMlG4ehiFQ0d4vSKrjriHsLQuSJOThYUl9AAgtBx+ZPcdGytmFCBt7iAqt+X/GOa/RKtjKZAfekDcF1tPcUydyqODOWhLB7MA2V8DrTbkuv3TS76ZRA101hYhlHCRcexKtQk3chhzLOZwkJIENhJlyjwz3kNEH8Z90gvME4iQLb4cmpy9Y8cIHdCQgR1xTjgdD6W9SH52tLF3bvqU6OGLSR0iZeOeAkpSTH0LlqdrIplZApUcTaqovbLdaZEe5MUNSgOA2+jFwjRuxkB14Qq2qB1m6BL2Ei4N/lwL88RwgSaN0+F6svRHKIc4Wb2aPp3o9jemQQpCjTCwr5C7UCLIJ5nzVpvgQ8eP3G/HaKlcVHZqwcAepJbJegZ4BXhpP1HRwefAgcf7HQ2wj/oxtt0iTaeRTKDiNoB23OPypxJ0Dlu2Is7PkgFbT7dT3FmwinPBhPP9Wr0cBaxZynnmZOpcpbcwNlrsZuZmyp3Wi+sm6l5/+mu/8emf/tZnf2tu/qde+7WKDsmt2vzQUG10lKl4ZW7x5UY1V1qA+bpFUyrnZ2e7MURWys7JbHFSc6soxznu4QkkNNFbr+8LxVJQZqZnB+dia4OWdlgBA52ld9StF8uS/wwviRbAqd4UxzxmyRnSRQ9UtACbfNH8Wd9KyZsPHAEqZrZSOzgMY1R2DyZE8atgJzmn0ljjwEZ9f3tz69HHL+Ghc/NFvccgJL1Hs518qSj7GYNkfrm3NkGzMx0eMo4DfikpWgY0UdZHRop4NCbm+OJZ7V6pnLcjDhrBId4FZnqgn8yZCDhObm7vdaIvpeQYMcUWrEAF+DjfLkXBtZznYsSmxkT63IhOHRWJZjM2n5+CTm5lMfarTobejqLl3gIdVoMMnSkE4wuk5sMQz4NXGOCArKCeCzAGV3IKMqvEqnTqhtQKXhhXswvLGgKub21KtyiWFZpKOwiEDaQS7MtF7ew4Bmjqy2SGgZHNCIKGQ8YwH1YwelNiqLiK5qy9WjJnudm7WJwlWDaLKPBcX61UWww5pXzpKB5eJZtVuDyEJ0S/C/EWmB5ZNdK+xmXFTpp+EUgbPUAk6jZzcvUhQWKkt26+BbHzE/mxWGN+7WjDQx32ypJh3kcS7hQfyJpeSlW2wIiN6I8G5lTAqMrRgjqyuuQIl1D0/sEusgJPoKaLiIGLbTpigEAstCvsxC7Enqu1bYClJaBFIgBUfcVZRKwNi5wgm3dsytrcln/LfhGygL9xSZHLcHRU1DYj2IIGVGHX+frA1+09xj588fKiRK8HaxvIQG6xw3vXu9/99Re/xqlBMxK/cR3gDlR4T5K6KSU2WGSqAzl38QIAcSnPzleie5wurLmypZOdhiihEOnQ+Is9HOzt+RyiU7ve+fTTmlPKecK//s2/+Tfvec+z/Phf/foLkr8NvuUT1rJDMpde5DFoSM54snMCL8cyRMsj166xjL15z7vejRGw7SzSCwi4H/GhYHcnR0aagfLK0oqjf/pd75E3bGs6i2o0k5zklzW9kl762d/57fe+972rd24zcKEgNMV/rU3LOlsjxcBUdyq8MvwGRPvpcb1V5/JEVJYEJpAGPnm2NdjpIBVQJNXTwxAD5hAxEWbgKUJw6URlN0TKAw8kPEBO1IcAWqloWrDMuUD9eMXYXeTBsEPvaJEmqDMA4PsqnWmAQwSwnNZ8YUoJug527u/RXlBBfMCq/pQ+xQ79aoMwZnANB6zFcV5CeoRxok+v7i9J+WVKKiQgtJxFoC/lmn98coJ
```

Comment:
what is the source of this image?
We prefer do not store data in repo and download instead.